### ISSUE TITLE
Is it possible to initialize custom title, image, album,... in RxMusicPlayerItem?

### ISSUE THREAD

**Issue by dinhthiet2702** [yoheimuta/RxMusicPlayer#61]
Some of my songs don't have image, title and album metadata, can I customize it? I mean will still initialize with the url, but may have other metadata, if not manually added will read the metadata of the song.

**Comment by yoheimuta** [comment]
@dinhthiet2702 Yes. It's possible to initialize RxMusicPlayerItem with custom metadata.
See the below links for details.

- https://github.com/yoheimuta/RxMusicPlayer/pull/42/files#diff-bbc433fa04f4ca1c593396321c6772fe9c6ede6b597d3813a0f2ff6925f879e3R46
- https://github.com/yoheimuta/RxMusicPlayer/issues/39#issuecomment-937358196
